src/DKCTropicalFreeze/Graphics/rules.txt

```diff
@@ -0,0 +1,341 @@
+[Definition]
+titleIds = 0005000010137F00,0005000010138300,0005000010144800
+name = Graphics Settings
+path = "Donkey Kong Country: Tropical Freeze/Graphics"
+description = Changes the resolution of the game.|Made by getdls and Skalfate.
+#Credits: getdls, Skalfate
```

Comment:
The credits comment may look unnecessary on this file because the credits are already in the description
but the other DKC mods have a credits comment, and in my v7 draft, every mod has a credits comment
so I'm adding it for consistency

this is gonna be the case for a lot of mods going forward so just assume reasons above is you see what looks like an unnecessary credits comment